pkg/apis/cr/v1/backup.go

```diff
@@ -18,6 +19,9 @@ type MySQLBackup struct {
 
 // MySQLBackupSpec stores the properties of a MySQL cluster backup.
 type MySQLBackupSpec struct {
+	Cluster string            `json:"cluster"`
+	Time    string            `json:"time"`
+	Storage resource.Quantity `json:"storage"`
```

Comment:
We should keep a list of backups already created with this Spec to easily restore a cluster from specific file.

For example keep a list of such entries (update during cronjob)
```go
type MySQLBackupInstance {
        CreatedAt string `json:"createdAt"`
        Filename  string `json:"filename"`
        Status    string `json:"status",omitempty`
}

Reply:
We'll provide it as a list of events. To be done in next pull requests.